src/components/windows-motion-controls.js

```diff
@@ -319,36 +318,36 @@ module.exports.Component = registerComponent('windows-motion-controls', {
         }
       }
 
-      // Calculate the pointer pose (used for rays), by applying the inverse holding pose, then the pointing pose.
+      // Calculate the pointer pose (used for rays), by applying the world transform of th POINTER_POSE node
+      // in the glTF (assumes that root node is at world origin)
```

Comment:
Can we move the logic to calculate the ray origin to a separate function? The onModelLoaded function is getting pretty big